.flake8

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+
+[flake8]
+ignore =
+  E111  # indentation is not a multiple of four
```

Comment:
**Note to reviewers**: I would like to submit a separate pull request that fixes all linter issues.

Reply:
FWIW, I don't really like the 4 space indentation from PEP-8, so most of this is intentional.

Reply:
No problem.  I might be able to fix some of the others though.